res/test_update_metadata_cli.py

```
@@ -0,0 +1,86 @@
+                    text=True,
+                    capture_output=True,
+                )
+                self.assertIn("the following arguments are required", result.stderr)
+                self.assertIn("--package-id", result.stderr)
+                self.assertIn("--release-base-url", result.stderr)
+
+    def test_sign_rejects_empty_artifact(self):
+        with tempfile.TemporaryDirectory() as directory:
+            artifact = Path(directory) / "rustdesk-1.4.6-aarch64.dmg"
+            artifact.touch()
+            result = subprocess.run(
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)